src/nomad_material_processing/vapor_deposition.py

```diff
@@ -178,7 +173,7 @@ class SubstrateTemperature(ArchiveSection):
     )
     temperature = Quantity(
         type=float,
-        unit="kelvin",
+        unit="kelvin", ########### the unit can change from user to user ###############
```

Comment:
How do you mean? The SI unit of temperature is kelvin, what would you propose instead?
```suggestion
        unit="kelvin",
```